managed/models/database.go

```diff
@@ -29,10 +30,16 @@ import (
 	"google.golang.org/grpc/status"
 	"gopkg.in/reform.v1"
 	"gopkg.in/reform.v1/dialects/postgresql"
+
+	"github.com/percona/pmm/agent/utils/version"
```

Comment:
don't import `agent` packages into `managed`

Reply:
if you would like to re-use something, extract it from there

Reply:
managed model for postgres versioner added